packages/kmod-6.1-nvidia-r570/nvidia-tmpfiles.conf.in

```diff
@@ -0,0 +1,7 @@
+R __PREFIX__/lib/modules/__KERNEL_VERSION__/kernel/drivers/extra/video/nvidia/tesla - - - - -
+d __PREFIX__/lib/modules/__KERNEL_VERSION__/kernel/drivers/extra/video/nvidia/tesla 0755 root root - -
+R __PREFIX__/lib/modules/__KERNEL_VERSION__/kernel/drivers/extra/video/nvidia/open-gpu - - - - -
+d __PREFIX__/lib/modules/__KERNEL_VERSION__/kernel/drivers/extra/video/nvidia/open-gpu 0755 root root - -
+C /etc/nvidia/fabricmanager.cfg - - - -
+d /run/nvidia 0700 root root -
+D /usr/bin/nvidia-persistenced 0755 nvidia nvidia - -
```

Comment:
This should be `/var/run/` not `/usr/bin`:

```suggestion
D /var/run/nvidia-persistenced 0755 nvidia nvidia - -
```

`systemd-tmpfilesd` won't fail but with this line you should see a warning in the journal.